solutions/usaco-987.mdx

```diff
@@ -53,4 +53,44 @@ int main(){
 
 </CPPSection>
 
+<PySection>
+
+## Implementation
+
+```python
+import sys
+
+sys.stdin = open("word.in", "r")
+sys.stdout = open("word.out", "w")
+input = sys.stdin.readline
+
+def bessie_word(n, k, t):
```

Comment:
literally no need to have this in a function

Reply:
okay ill change it

Reply:
fixed